### DISCUSSION TITLE
Παραδοτέο 3ο | Π2018122

### DISCUSSION THREAD

**Discussion by AndreasKarageorgiou** [courses-ionio/sw#1336] [Show and tell]
Για το 3ο παραδοτέο έπρεπε να εγκαταστήσουμε ένα σύστημα χωρίς systemd . Έπειτα από λίγο ψάξιμο κατέληξα στο Void Linux καθώς είναι αρκετά εύκολο και προσβάσιμο.
Παρακάτω βρίσκεται το **[Asciinema Link](https://asciinema.org/a/JpkJrZPmileotIc3zSoYW4aHQ)** που δημιούργησα.

**Comment by p20sami** [comment]
@AndreasKarageorgiou Στο command prompt, βάλε το ΑΜ σου όπως αναφέρεται στις [οδηγίες](https://epidrome.github.io/teaching/cli/)
